packages/thunderstore-api/src/fetch/teamAddMember.ts

```diff
@@ -0,0 +1,28 @@
+import { RequestConfig } from "../index";
+import { apiFetch2 } from "../apiFetch";
+
+export type teamAddMemberMetaData = {
+  teamIdentifier: string;
+};
+
+export type teamAddMemberApiArgs = {
+  user: string;
+  role: string;
+};
+
+export function teamAddMember(
+  config: RequestConfig,
+  data: teamAddMemberApiArgs,
```

Comment:
As mentioned in my review for #938 I'm not sure I like the split between "data" and "metadata". Especially for these small forms, might be a different story for a form with 10 fields, in which case having `data` (or `formData`) as a separate object might make sense. Even then I would pass whatever is in "metadata" as separate args.